packages/terra-disclosure-manager/README.md

```diff
@@ -0,0 +1,24 @@
+# Terra Disclosure Manager
+
+[![NPM version](http://img.shields.io/npm/v/terra-disclosure-manager.svg)](https://www.npmjs.org/package/terra-disclosure-manager)
+[![Build Status](https://travis-ci.org/cerner/terra-framework.svg?branch=master)](https://travis-ci.org/cerner/terra-framework)
+
+The DisclosureManager is a stateful component used to manage disclosure presentation.
+
+- [Getting Started](#getting-started)
+- [Documentation](https://github.com/cerner/terra-framework/tree/master/packages/terra-disclosure-manager/docs)
+- [LICENSE](#license)
+
+## Getting Started
+
+- Install from [npmjs](https://www.npmjs.com): `npm install terra-disclosure-manager`
+
+## LICENSE
+
+Copyright 2017 Cerner Innovation, Inc.
```

Comment:
Should we update our generator to use 2018?

Reply:
We'll need to talk to James on the open source committee to get a clear direction on this. I would assume that we update this to 2018. Though as this gets into legal territory, we'll need guidance on this. Not sure if we update to `2018` or to `2017 - 2018`.

Reply:
Logged https://github.com/cerner/terra-framework/issues/65 to follow up on this issue.